build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+    id 'java'
+    id 'idea'
```

Comment:
Out of curiosity, what are you using idea plugin for?

Reply:
I was certain that `idea` is needed to change source and target compatibility without changing Java SDK in InteliJ Project Structure. I tested without plugin and it works the same. For now I think I don't need this one.